.github/workflows/ci.yaml

```diff
@@ -168,10 +168,47 @@ jobs:
       - run: sudo apt install -y libpcap-dev cmake
       - run: cargo +nightly build --manifest-path=./Cargo.toml
 
+  integration_tests:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-12, macos-11]
+        target:
+          [x86_64-unknown-linux-gnu, x86_64-apple-darwin, aarch64-apple-darwin]
```

Comment:
not sure how building aarch64 helps here as we can't run integration tests of it?

Reply:
unless we run emulation (not sure we even can) there are no GitHub runners for Apple Sillicon..